prime-router/docs/playbooks/how-to-use-token-auth.md

```diff
@@ -0,0 +1,63 @@
+# How To Use FHIR Style Token Authentication
+
+This playbook is a set of commandline scripts meant to exercise and demonstrate the 3 steps in FHIR style authentication.
+
+A couple notes before we get started:
+
+- Note: To do step 3, you will need to create and place a TokenSigningSecret in docker-compose.yml for this to work.  Might as well do this now beforeyou go any further, so you don't have to restart your container.  This is easy - run the "main" program in ReportStreamSecretFinder.kt, and it will create one for you!
```

Comment:
```suggestion
- Note: To do step 3, you will need to create and place a TokenSigningSecret in docker-compose.yml for this to work.  Might as well do this now before you go any further, so you don't have to restart your container.  This is easy - run the "main" program in ReportStreamSecretFinder.kt, and it will create one for you!
```